bukuserver/views.py

```diff
@@ -49,6 +49,28 @@ def search(self):
         return redirect(url)
 
 
+def last_page(self):
+    """Generic '/last_page' endpoint handler"""
+    # Grab parameters from URL
+    view_args = self._get_list_extra_args()
+
+    # Map column index to column name
+    sort_column = self._get_column_by_idx(view_args.sort)
+    if sort_column is not None:
+        sort_column = sort_column[0]
+
+    # Get page size
+    page_size = view_args.page_size or self.page_size
+
+    # Get count and data
+    count, data = self.get_list(-1, sort_column, view_args.sort_desc,
+                                view_args.search, view_args.filters, page_size=page_size)
```

Comment:
The above code is taken from [here](https://github.com/flask-admin/flask-admin/blob/v1.6.0/flask_admin/model/base.py#L1956-L1969) (sans the page index)

Reply:
put the link on function docstring